Gemfile

```diff
@@ -1,23 +1,18 @@
 source 'https://rubygems.org'
 
-gem 'addressable', '2.3.5'      # pin to avoid RDF bug
-gem 'dor-services', '~> 4.8'
+# General ruby dependencies
+gem 'pry', '~> 0.10.0'           # for bin/console
+gem 'slop', '~> 3.5.0'           # for bin/run_robot
+gem 'rake', '~> 10.3'
+
+# DLSS gems
+gem 'dor-services', '~> 5.3'
 gem 'lyber-core', '~> 3.2', '>=3.2.4'
```

Comment:
Yes, SemVer allows the most specific component of the version number to fluctuate.  

For `~> 3.2.0`, upper bound (non-inclusive) is essentially `3.3`.  Any `3.2.x` is eligible.
For `~> 3.2`, upper bound (non-inclusive) is essentially `4.0`.  Any `3.x` is eligible, where `x>=2`. 

This is a slight gloss, because version depth can vary, e.g. `3.3.66.11betaQ` would also be eligible.  But you get the idea.